livy/livy.sh

```diff
@@ -107,6 +107,14 @@ function main() {
 
   wget -nv --timeout=30 --tries=5 --retry-connrefused "${LIVY_URL}" -P "${temp}"
 
+  local OS
+  OS=$(. /etc/os-release && echo "${ID}")
+  if [[ "${OS}" ==  "rocky" ]]; then
+    yum install -y unzip
+  else
+    sudo apt-get install -y unzip
```

Comment:
Is there a reason the `else` path needs `sudo` but not the `if` path? I'd expect `sudo` is unnecessary on either path, because initialization actions always run as root:

https://cloud.google.com/dataproc/docs/concepts/configuring-clusters/init-actions

> Initialization actions run as the **root** user. You **do not** need to use `sudo`.

Reply:
done